dank-client/client.go

```diff
@@ -37,6 +42,28 @@ func (d *Client) resolve() string {
 	return srvclient.MaybeSRV(d.hostname)
 }
 
+var quoteEscaper = strings.NewReplacer("\\", "\\\\", `"`, "\\\"")
```

Comment:
would `https://golang.org/pkg/strconv/#QuoteToASCII` do the same thing you're trying to do here?